doc/jemalloc.xml.in

```diff
@@ -1544,8 +1573,24 @@ malloc_conf = "xmalloc:true";]]></programlisting>
         set, all currently unused dirty pages are considered to have fully
         decayed, which causes immediate purging of all unused dirty pages unless
         the decay time is set to -1 (i.e. purging disabled).  See <link
-        linkend="opt.decay_time"><mallctl>opt.decay_time</mallctl></link> for
-        additional information.</para></listitem>
+        linkend="opt.dirty_decay_time"><mallctl>opt.dirty_decay_time</mallctl></link>
+        for additional information.</para></listitem>
+      </varlistentry>
+
+      <varlistentry id="arena.i.muzzy_decay_time">
```

Comment:
I went through several iterations on naming, and considered "fully" vs "partially" purged.  The main problem with those names is that they can easily be confused to mean that the pages within an extent are in a homogeneous vs heterogeneous state, but I also didn't consider the terms to precisely match the semantics.

Part of the difficulty is that we need names for both the states and the functions that cause state transitions:
- deallocate(active) --> dirty
- lazy_purge(dirty) --> muzzy
- forced_purge(muzzy) --> retained

Here's the list of alternative names I came up with:
- dingy
- unknown
- purgatory
- limbo
- hazy
- fuzzy
- muzzy

"Muzzy" actually has the correct definition; its only detraction is that it might sound like a made up word due to being uncommonly used.